_2022_pages/agenda.html

```diff
@@ -26,12 +26,12 @@ <h2><a href="https://wiki.hyperledger.org/display/events/Ideate+Challenge">Ideat
                 <td><a href="https://wiki.hyperledger.org/display/events/Submissions">Submit</a></td>
             </tr>
             <tr>
-                <td>01 March 2022</td>
+                <td>15 March 2022 <br>(31 March for China)</td>
                 <td>Ideate submission ends</td>
                 <td></td>
             </tr>
             <tr>
-                <td>01 March 2022 to 15 March 2022</td>
+                <td>01 March 2022 to 15 March 2022<br> (15 March to 31 March for China)</td>
```

Comment:
Evaluation for China would end on 31st March but it can happen even before 15th. So maybe this can be updated to reflect that evaluation starts on 1st and ends on 15th March for global submissions, 31st March for China region.